consensus/src/commons.rs

```diff
@@ -139,7 +139,6 @@ impl From<BlsSigError> for ConsensusError {
 #[async_trait::async_trait]
 pub trait Database: Send + Sync {
     fn store_candidate_block(&mut self, b: Block);
-    fn delete_candidate_blocks(&mut self);
```

Comment:
This should not be removed

If it's not used, it's a bug

Indeed, we currently store candidates during the selection phase, and those candidates are supposed to be removed while accepting a block

Reply:
Ok. I'll revert this change and create a new issue to fix the bug

Reply:
As pointed out by @goshawk-3 the delete_candidate_blocks is indeed unused now, since the candidates are deleted using `Candidate::delete` function